docs/central-entities.rst

```diff
@@ -233,6 +233,107 @@ You can see the full XLSForm `here <https://docs.google.com/spreadsheets/d/12oJZ
+
+**All** conflicts surfaced by Central occur because two Entity updates were generated based off the same base version of an Entity. This circumstance is called a :guilabel:`Parallel Update` in Central. In the special case that a property included in an update has also been changed in the meantime, *and* the value currently indicated in Central is different from the new updated value, a full :guilabel:`Conflict` error will be created instead, indicating that multiple possibly conflicting values were written into the same property at once.
+
+.. _central-entities-build-update:
+
+Building a Form that updates an Entity
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Updating Entities isn't too different from creating new Entities, but you'll have to specify the ``entity_id`` of the specific Entity instance you want to update in the ``entities`` sheet.
+
+.. rubric:: XLSForm
+
+.. csv-table:: entities
+  :header: list_name, entity_id
+ 
+  trees, ${tree}
+
+In this example, a tree in the ``trees`` Entity List will be updated when Submissions based on this Form are created. The particular tree to be updated must be given in the ``entity_id`` column, which here is an expression bound to the Submission that computes to the unique identifier of that tree for each data update.
+
+Once you have this update information set up, you can add ``save_to`` rules to your Form questions as with Entity creation:
+
+.. csv-table:: survey
+  :header: type, name, label, save_to
+ 
+  geopoint, current_location, Tree location, location
+  select_one species, species, Tree species
+  integer, circumference, Tree circumference in cm, circumference_cm
+  text, intake_notes, Intake notes
+
+Here, the ``current_location`` and ``circumference`` questions have been set up to update properties in the Entity, while the ``species`` and ``intake_notes`` questions don't update anything to the Entity.
+
+.. _central-entities_build-update-label:
+
+Updating the Entity Label
+"""""""""""""""""""""""""
+
+To update the label of an Entity from a Form, fill in the optional ``label`` column in the ``entities`` sheet as you would when setting the label on Entity creation.
+
+.. rubric:: XLSForm
+
+.. csv-table:: entities
+  :header: list_name, entity_id, label
+ 
+  trees, ${tree},"concat(${circumference}, ""cm "", ${species})"
+
+.. _central-entities_build-update-conditional:
+
+Set conditions under which an Entity is updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can limit a Form to only update an Entity under certain conditions, based on an expression using the Entity and Submission data.
+
+.. rubric:: XLSForm
+
+.. csv-table:: entities
+  :header: list_name, entity_id, update
+ 
+  trees, ${tree}, true()
+
+In this case, ``true()`` is given in the optional ``update`` column, which means the Entity will always be updated upon submission. If an expression is given instead, the Entity is only updated when the expression evaluates to ``true`` or ``1``.
```

Comment:
This looks good to me. The kind of application I can think of for `update_if` is a form that has a section with the option to update certain entity properties and a select to make that section relevant. If there are no properties to update, might as well not send an update request. That feels like it needs some setup to explain and I think what you have gets the concept across.